doc/tutorials/pick_and_place_with_moveit_task_constructor/launch/mtc_demo.launch.py

```diff
@@ -12,6 +12,9 @@ def generate_launch_description():
         MoveItConfigsBuilder("moveit_resources_panda")
         .robot_description(file_path="config/panda.urdf.xacro")
         .trajectory_execution(file_path="config/gripper_moveit_controllers.yaml")
+        .planning_scene_monitor(
```

Comment:
I got errors when I tried to run the tutorials without this

Reply:
Can you include the error logs here?

Reply:
[mtc_node-1] [WARN] [1656956892.458226104] [moveit_ros.planning_scene_interface.planning_scene_interface]: service '/get_planning_scene' not advertised yet. Continue waiting...

Reply:
Doesn't seem like an issue when I built it outside of docker. In a future PR I'll test everything in a container so I can confirm the change is not needed